packages/block-library/src/tag-cloud/index.php

```diff
@@ -57,7 +57,6 @@ function register_block_core_tag_cloud() {
 		array(
 			'name'         => 'outline',
 			'label'        => __( 'Outline', 'gutenberg' ),
-			'style_handle' => 'outline',
```

Comment:
You should be able to remove `register_block_style` completely now.

Reply:
Oh I see, thanks that's neat.